Makefile

```diff
@@ -83,7 +83,7 @@ TESTER_NAME = tester
 si:
 	@echo "Installing from: $(PROJECT_INSTALL)"
 ifeq ($(PROJECT_INSTALL), config)
-	$(call php, drush si config_installer --db-url=$(DB_URL) --account-name=$(ADMIN_NAME) --account-mail=$(ADMIN_MAIL) --account-pass=$(ADMIN_PW) -y config_installer_sync_configure_form.sync_directory=../config/sync)
+	$(call php, drush si --existing-config --db-url=$(DB_URL) --account-name=$(ADMIN_NAME) --account-mail=$(ADMIN_MAIL) --account-pass=$(ADMIN_PW) -y)
```

Comment:
Iirc it doesn't work with profile other then minimal

Reply:
there is also suggestion to use hook_install_tasks and may be help with https://www.drupal.org/project/drupal/issues/2982052 so we would be able to use own profile.

Reply:
I had the reference to issue https://www.drupal.org/project/drupal/issues/2982052 on code.
Also, I've tested it this way and it is working fine with custom profile

Reply:
Confirmed in QA test